test_relationship_server.py

```diff
@@ -0,0 +1,146 @@
+import pytest
+
+from relationship_server import record_processor
+
+
+def test_record_processor():
```

Comment:
Add tests for missing keys with a patient id.

Reply:
solved by 78cbd9b